shard.yml

```diff
@@ -1,7 +1,7 @@
-name: fast-http-server
-version: 0.4.0
-
-authors:
-  - Serdar Dogruyol <dogruyolserdar@gmail.com>
-
-license: MIT
+name: fast-http-server

```

Comment:
Do you know of a good way to fix all line endings? I already tried to change them to unix style but some still have it